src/RuleSet.php

```diff
@@ -56,12 +56,53 @@ final class RuleSet implements RuleSetInterface
             'switch_case_space' => true,
             'visibility_required' => true,
         ],
-        '@Symfony' => [
+        '@PSR12' => [
             '@PSR2' => true,
-            'array_syntax' => ['syntax' => 'short'],
-            'backtick_to_shell_exec' => true,
             'binary_operator_spaces' => true,
```

Comment:
This is too strict. The spec says at least one space on the right, and not exactly one.

Reply:
I think we need to add another TODO to do the psr-12 compat issue.

Reply:
Removed.